### ISSUE TITLE
Space-Efficient Display of Many Author Affiliations 

### ISSUE THREAD

**Issue by peitek** [gi-ev/LNI#66]
Hi,

I'm using the template to submit a paper, but ran into a minor issue. Our paper has nine authors overall, which come from different, but partly overlapping institutions. We do not share the author's email addresses, and only the institutions. Unfortunately, with the default styling of one footnote per author, this seems to waste a lot of space. 

My `\author` section currently looks like this:

```
\author[Vorname1 Nachname1 \and Firstname2 Lastname2 \and Firstname3 Lastname3 \and Firstname4 Lastname4 \and Firstname5 Lastname5 \and Firstname6 Lastname6 \and Firstname7 Lastname7 \and Firstname8 Lastname8 \and Firstname9 Lastname9]
{Vorname1 Nachname1\footnote{Universität 1, Abteilung, Straße, Postleitzahl Ort, Land \email{emailaddress@author1}} \and
Firstname2 Lastname2\footnote{University 2, Department, Address, Country
\email{emailaddress@author2}} \and
Firstname3 Lastname3\footnote{University 2, Department, Address, Country
\email{emailaddress@author3}} \and
Firstname4 Lastname4\footnote{University 2, Department, Address, Country
\email{emailaddress@author4}} \and
Firstname5 Lastname5\footnote{Universität 1, Abteilung, Straße, Postleitzahl Ort, Land \email{emailaddress@author5}} \and
Firstname6 Lastname6\footnote{University 2, Department, Address, Country
\email{emailaddress@author6}} \and
Firstname7 Lastname7\footnote{University 2, Department, Address, Country
\email{emailaddress@author7}} \and
Firstname8 Lastname8\footnote{Universität 1, Abteilung, Straße, Postleitzahl Ort, Land \email{emailaddress@author7}} \and
Firstname9 Lastname9\footnote{Universität 1, Abteilung, Straße, Postleitzahl Ort, Land \email{emailaddress@author9}}}
```

This is the resulting pdf: [lni-author-template.pdf](https://github.com/gi-ev/LNI/files/1487599/lni-author-template.pdf).

Is there a way to style the author affiliations more space efficient? For example, in our case multiple authors could (theoretically) share an affiliation footnote.

Thank you for your help!

**Comment by koppor** [comment]
This is the output:

![grafik](https://user-images.githubusercontent.com/1366654/33139327-3ea90688-cfad-11e7-88df-ef6938911499.png)

The current solution is that the authors tweak the rendering manually. That is, they use `\footnote` manually. However, I do not know, how efficiently that currently works.

**Comment by koppor** [comment]
@peitek You are very welcome to make a proposal for a good latex command interface and good rendering.

**Comment by peitek** [comment]
@koppor Thanks for the reply! I don't know latex well, so I don't really have a good solution for it or could come up with an idea how it should be.

I ended up slapping something together with `\footnote` and a manual assignment of the footnote numbers with superscript. That seems to work fine, so you can close this issue, if you want. 

Thanks again for your quick help.

**Comment by koppor** [comment]
@peitek Yes, this is the normal procedure. However, this should also be documented, therefore I leave this issue opened. 😇

**Comment by stefan2904** [comment]
I think it makes more sense to use `\footnotemark` instead of superscript (as described [here](https://tex.stackexchange.com/a/35044/47460)), but it LaTeX throws errors if you do this in the LNI template ...

**Comment by koppor** [comment]
I think, a proper "howto" would also suffice to solve the issue. #documentation

**Comment by koppor** [comment]
I think, this is solved with v1.8. See

![image](https://github.com/gi-ev/LNI/assets/1366654/f236dcfa-9f20-4f6f-8d2a-0a8574786136)

Please comment if this does not completely solve the issue.
